### PR TITLE
provide more access to deployment details in post deployment step

### PR DESCRIPTION
**Pull request by pascalism**

### CODE REVIEW

**Comment by microwaves** [comment]
Cool feature idea @pascalism. Could you please add some tests? Cheers

File: README.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ Usage example:
 Where `app-backend` stands for the application name, `app-backend-container` for the container name,
 `cluster1` for the cluster name and `v1.0.1` for the desired image tag.
 
+#### Post Deployment Steps
+
+You can define post deployment steps that enable you to execute a command after a successful deployment, e.g. testing. 
+Also, you can access username, component name and cluster of your deployment within the command with the help of Go's [template package](https://golang.org/pkg/text/template/#Template.DefinedTemplates).
+For instance the username, which could be accesses like this: `curl --foo:{{.Username}}`
```
Comment (jmarten) [inline README.md]: here we should also add how the component name and the cluster is referenced.

File: ext/kubernetes/deployment/init.go [hunk 2]
```diff
@@ -1,7 +1,9 @@
 package deployment
 
 import (
+	"bytes"
 	"fmt"
+	"html/template"
```
Comment (jmarten) [inline ext/kubernetes/deployment/init.go]: Could we actually use text/template instead of html/template?

Comment (pascalism) [reply ext/kubernetes/deployment/init.go]: Yes, and we should, I guess. 👍

File: ext/kubernetes/deployment/init_test.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	}
+	if error != nil {
+		t.Fatal("Error during post deployment info retrieval")
+	}
+}
+
+func TestGetPostDeploymentStepCommandUnsuccessful(t *testing.T) {
+	var commandTemplate = "curl"
+	command, _ := addDeploymentInfoToCommand(commandTemplate, "Gryffindor", "Hogwarts", "Harry Potter")
+	if strings.Contains(command, "Harry Potter") {
```
Comment (jmarten) [inline ext/kubernetes/deployment/init_test.go]: This check can actually be removed, as the next equal check already ensures this.

**Pull request merged**